kwc-building-listing.html

```diff
@@ -8,27 +10,136 @@
 -->
 
 <dom-module id="kwc-building-listing">
-  <template>
-    <style>
-      :host {
-        display: block;
-      }
-    </style>
-    <h2>Hello [[prop1]]!</h2>
-  </template>
-
-  <script>
-    Polymer({
-
-      is: 'kwc-building-listing',
-
-      properties: {
-        prop1: {
-          type: String,
-          value: 'kwc-building-listing',
-        },
-      },
-
-    });
-  </script>
-</dom-module>
+    <template>
+        <style>
+             :host {
+                --paper-spinner-color: var(--color-kano-orange);
+                --kwc-challenge-card-cover: {
+                    height: 176px;
+                }
+            }
```

Comment:
Indentation might have gone a bit funky here – looks like the closing brace is further out than the opening tag

Reply:
Seems there is an extra space in from of the selector names, probably from deleting all the :host selectors with multiple cursor!